### PR TITLE
[IMP] inventory: overhaul 3rd party shippers

### PR DESCRIPTION
**Pull request by Felicious**
## Summary of changes
1. Overhaul 8 year old 3rd party shipping carrier doc to 16.0
2. Addresses @moch-odoo's [suggestions](https://www.odoo.com/web#id=3576549&cids=3&menu_id=4720&action=333&active_id=3835&model=project.task&view_type=form) to update shipping connector installation instructions
3. Add a table of shipping methods and region availability in Odoo  

## FW port
Yes, all the way to master (:

[Task](https://www.odoo.com/web#id=3576549&cids=3&menu_id=4720&action=333&active_id=3835&model=project.task&view_type=form)

### CODE REVIEW

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/third_party_shipper.rst
Comment (brse-odoo) [inline content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/third_party_shipper.rst:95]: For these instructions here, do you think some additional instruction on what these are, how you find them, and what they affect would be helpful? I know it's different per carrier, but maybe a .. note:: or a .. tip:: saying "read the doc for individual carriers to find this stuff" would be nice

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/third_party_shipper.rst
Comment (brse-odoo) [inline content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/third_party_shipper.rst:104]:
```suggestion
   Setting the delivery method to :guilabel:`Production` charges customers for shipping. Verify all
```

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/third_party_shipper.rst
Comment (brse-odoo) [inline content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/third_party_shipper.rst:105]:
```suggestion
   configurations are correct before launching the delivery method to :guilabel:`Production`.
```

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/third_party_shipper.rst
Comment (brse-odoo) [inline content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/third_party_shipper.rst:149]:
```suggestion
Then, switch to the :guilabel:`Inventory` tab, and define the :guilabel:`Weight` of the product, in
```

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/third_party_shipper.rst
Comment (brse-odoo) [inline content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/third_party_shipper.rst:150]: Why kilograms? Does it have to be? Can different products have different UoMs?

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/third_party_shipper.rst
Comment (brse-odoo) [inline content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/third_party_shipper.rst:164]: Or, we can create a new quotation also right?

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/third_party_shipper.rst
Comment (brse-odoo) [inline content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/third_party_shipper.rst:176]:
```suggestion
In the resulting pop-up window, select the intended carrier from the :guilabel:`Shipping Method` drop-down
```

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/third_party_shipper.rst
Comment (brse-odoo) [inline content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/third_party_shipper.rst:199]:
```suggestion
:guilabel:`Carrier` field. When the delivery method is set to :ref:`production mode
```

**Review by brse-odoo — APPROVED** [review]
@Felicious I've finished my review of this PR, great job! I'm approving with a few comments for you to accept/reject as you see fit. Let me know if you have any questions, thanks!

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/third_party_shipper.rst
Comment (Felicious) [inline content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/third_party_shipper.rst:95]:
```suggestion
of the shipping carrier.  For specific details about each
shipping method, refer to the following documents:

.. seealso::
   - :doc:`DHL credentials <dhl_credentials>`
   - :doc:`Sendcloud credentials <sendcloud_shipping>`
   - :doc:`UPS credentials <ups_credentials>
```

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/third_party_shipper.rst
Comment (Felicious) [reply content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/third_party_shipper.rst]: You're right! I confirmed with an expert, and this field has become really flexible since 8 years ago, when the line was written (:

**Comment by Felicious** [comment]
Hi @odoo/us-doc-review ! This doc is ready for review :smile:

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/third_party_shipper.rst
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/third_party_shipper.rst:227]:
```suggestion
delivery order, by first going to the :menuselection:`Inventory` app. Then, from the :guilabel:`Inventory Overview` dashboard, select the :guilabel:`Delivery
```

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/third_party_shipper.rst
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/third_party_shipper.rst:228]: do you think it would be helpful to mention that the user should choose a DO that isn't already marked as 'Done' -- or do you think that is redundant info?

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/third_party_shipper.rst
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/third_party_shipper.rst:230]:
```suggestion
In the :guilabel:`Additional info` tab, set :guilabel:`Carrier` field to the desired third-party shipping
```

**Review by ksc-odoo — CHANGES_REQUESTED** [review]
Hey @Felicious -- just finished my Final Review. Great work on this so far. I think we're *just* about at the finish line. Once you review my feedback, and make the necessary adjustments, feel free to tag me again for another quick look. Thanks! 👍

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/third_party_shipper.rst
Comment (Felicious) [reply content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/third_party_shipper.rst:9]: Very nice rewrite, thank you!

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/third_party_shipper.rst
Comment (Felicious) [reply content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/third_party_shipper.rst:93]: These are good points! Taking note for myself here: upon selecting a Provider, the following fields appear:

Integration Level
Invoicing Policy
Insurance Percentage
Configuration tab

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/third_party_shipper.rst
Comment (Felicious) [inline content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/third_party_shipper.rst:130]:
```suggestion
set it to :guilabel:`Production Environment`.
```

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/third_party_shipper.rst
Comment (Felicious) [inline content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/third_party_shipper.rst:204]:
```suggestion
drop-down menu. The :guilabel:`Cost` field is automatically filled based on:
- the amount specified in the :guilabel:`Total Order Weight` field (if it is not provided, the sum of :ref:`product weights <inventory/shipping_receiving/configure-weight>` in the order is used) 
- the distance between the warehouse's :ref:`source
```

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/third_party_shipper.rst
Comment (Felicious) [inline content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/third_party_shipper.rst:209]:
```suggestion
After selecting a third party provider in the :guilabel:`Shipping Method` field, click :guilabel:`Get Rate` in the :guilabel:`Add a shipping method` pop-up window to get the
```

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/third_party_shipper.rst
Comment (Felicious) [inline content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/third_party_shipper.rst:228]:
```suggestion
Orders` operation type, and choose the desired delivery order that is not already marked as :guilabel:`Done` or :guilabel:`Cancelled`.
```

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/third_party_shipper.rst
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/third_party_shipper.rst:43]:
```suggestion
To ensure proper setup of a third-party shipping carrier with Odoo, follow these steps:
```

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/third_party_shipper.rst
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/third_party_shipper.rst:154]:
```suggestion
   are at risk of being charged through their carrier account (e.g. UPS, FedEx, etc.) **before**
```

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/third_party_shipper.rst
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/third_party_shipper.rst:178]:
```suggestion
Verify that the :guilabel:`Address` and :guilabel:`Phone` number are correct, as they are required for
```

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/third_party_shipper.rst
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/third_party_shipper.rst:214]:
```suggestion
To assign a third-party shipping carrier, and get an estimated cost of shipping, begin by going to
```

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/third_party_shipper.rst
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/third_party_shipper.rst:234]:
```suggestion
After selecting a third-party provider in the :guilabel:`Shipping Method` field, click
```

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/third_party_shipper.rst
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/third_party_shipper.rst:258]:
```suggestion
In the :guilabel:`Additional info` tab, set the :guilabel:`Carrier` field to the desired third-party
```

**Review by ksc-odoo — APPROVED** [review]
Okie dokie, @Felicious -- after you address the few comments I've left, this should be ready to be tagged for Tech Review. Approving now! 👍

**Comment by Felicious** [comment]
Hello @samueljlieber ! This PR is ready for your technical review :smile: Thank you, as always!

**Comment by Felicious** [comment]
Hi @samueljlieber ! Thank you so much for reminding me about the media file convention -- totally forgot about to reference it. Appreciate your attention to detail, as always! :grin: 

I've addressed all your comments, and the PR's ready for a second review

**Review by samueljlieber — APPROVED** [review]
Nice job @Felicious, your changes look good to me! Nice job :)
.....
@robodoo r+

**Pull request closed without merging**
